Libraries/Brawl/IT/BaseItem.h

```diff
@@ -2,6 +2,7 @@
 #ifndef PROJECTMCODES_BASEITEM_H
 #define PROJECTMCODES_BASEITEM_H
 
+#include "stdtypes.h"
```

Comment:
Was this needed to fix a compilation error - same question for itManager.h

Reply:
Yes.